client/src/pages/DashBoard.js

```diff
@@ -11,13 +11,23 @@ import UserInformation from '../components/UserInformation';
 import { createInterview } from '../utils/apiFunctions';
 import { CustomButton } from '../components/Buttons';
 import { Dialog, DialogTitle } from '@material-ui/core';
+import SocketContext from '../context/socket';
 
 const DashBoard = () => {
   const classes = useStyles();
   const history = useHistory();
   const [open, setOpen] = React.useState(false);
   const [selectedValue, setSelectedValue] = React.useState('Medium');
   const { state } = useContext(store);
+  const socket = useContext(SocketContext);
+
+  useEffect(() => {
+    socket.emit('logged_in', state.user._id);
+  }, [state.user, socket]);
+
+  useEffect(() => {
+    socket.emit('check_rooms', { userId: state.user._id });
```

Comment:
why do you check_rooms here?

Reply:
No longer there

Reply:
Was fixed when I did the merge conflicts